src/js/components/Box.js

```diff
@@ -153,6 +159,7 @@ Box.propTypes = {
   alignContent: PropTypes.oneOf(['start', 'center', 'end', 'between', 'around', 'stretch']),
   appCentered: PropTypes.bool,
   backgroundImage: PropTypes.string,
+  children: PropTypes.any,
```

Comment:
Children property was declared so it is on the list to be omitted for the prop transfer.